src/pages/ecosystem/contributing.md

```diff
@@ -219,7 +227,7 @@ Which will render:
 
 (define-public (test-emit-event)
     (begin
-        (print "Event! Hello world")
+        (print "Event. Hello world")
```

Comment:
This could be a!

Reply:
this is an interesting one because it should by default skip all sample codes. Must be because we have a docs and clarity specific annotation there: "```clarity highlight=1,4-6,13-17,28-32".

As opposed to spending more time, I'd rather just change up the print command to keep things simple for now